src/Core/Prefab/Globe/MiniGlobe.js

```diff
@@ -0,0 +1,86 @@
+import * as THREE from 'three';
+import { createGlobeLayer } from '../GlobeView';
+import View from '../../View';
+import SubdivisionControl from '../../../Process/SubdivisionControl';
+import { globeSubdivisionControl } from '../../../Process/GlobeTileProcessing';
+import { processTiledGeometryNode } from '../../../Process/TiledNodeProcessing';
+
+function TargetMesh(layer) {
+    const ring = new THREE.RingGeometry(40, 37, 32);
+    const point = new THREE.CircleGeometry(2, 32);
+    const shadow = new THREE.CircleGeometry(5, 32);
+    const material = new THREE.MeshBasicMaterial({ color: 0xcccccc, side: THREE.DoubleSide, opacity: 0.8, transparent: true });
+    const shadowMesh = new THREE.Mesh(shadow, new THREE.MeshBasicMaterial({ color: 0x000000, side: THREE.DoubleSide, opacity: 0.5, transparent: true }));
+    const targetMesh = new THREE.Mesh(ring, material);
+    const pointMesh = new THREE.Mesh(point, new THREE.MeshBasicMaterial({ color: 0xffffff, side: THREE.DoubleSide, opacity: 0.85, transparent: true }));
+    targetMesh.add(pointMesh);
+    targetMesh.position.set(0, 0, -500);
+    shadowMesh.translateZ(-10);
+    targetMesh.add(shadowMesh);
+    targetMesh.layers.set(layer);
+    shadowMesh.layers.set(layer);
+    pointMesh.layers.set(layer);
+
+    return targetMesh;
```

Comment:
Is this mesh supposed to indicate on the mini-globe the visible area in the big one? 
Because here it doesn't really match (the area displayed in the big globe is bigger than the area indicated by this mesh)